tests/ui/four_forward_slashes.rs

```diff
@@ -0,0 +1,44 @@
+//// first line borked doc comment. doesn't combust!
```

Comment:
I think we'll need a separate test file for this, as we can't have `#![...]` below items

Reply:
Yeah, sounds reasonable :)